### PR TITLE
example to show the 3d dataset

### PR DESCRIPTION
**Pull request by emmanuelle**
This new gallery example shows the 3d dataset which has been recently introduced.

See for example https://1138-2014929-gh.circle-artifacts.com/0/doc/build/html/auto_examples/data/plot_3d.html#sphx-glr-auto-examples-data-plot-3d-py

### CODE REVIEW

File: doc/examples/data/plot_3d.py
Comment (mkcor) [inline doc/examples/data/plot_3d.py:3]:
```suggestion
Datasets with 3 or more spatial dimensions
```
(all gallery examples have capitalized titles)

File: doc/examples/data/plot_3d.py
Comment (mkcor) [inline doc/examples/data/plot_3d.py:6]:
```suggestion
Most scikit-image functions are compatible with 3D datasets, i.e., images with
```
(current docs spell "3D" rather than "3-D")

File: doc/examples/data/plot_3d.py
Comment (mkcor) [inline doc/examples/data/plot_3d.py:7]:
```suggestion
3 spatial dimensions (to be distinguished from 2D multichannel images, which are also arrays with
```
Sorry for breaking the line length...

File: doc/examples/data/plot_3d.py
Comment (mkcor) [inline doc/examples/data/plot_3d.py:8]:
```suggestion
three axes). :func:`skimage.data.cells3d` returns a 3D fluorescence microscopy
```

File: doc/examples/data/plot_3d.py
Comment (mkcor) [inline doc/examples/data/plot_3d.py:10]:
```suggestion
provided in ``(z, c, y, x)`` order. Channel 0 contains cell membranes, while channel
```

File: doc/examples/data/plot_3d.py
Comment (mkcor) [inline doc/examples/data/plot_3d.py:9]:
```suggestion
image of cells. The returned dataset is a 3D multichannel image with dimensions
```
Alternatively, it's a 4D array (like, `img.ndim`)... right?

File: doc/examples/data/plot_3d.py
Comment (mkcor) [inline doc/examples/data/plot_3d.py:13]:
```suggestion
The example below shows how to explore this dataset. This 3D image can be used
```

File: doc/examples/data/plot_3d.py
Comment (mkcor) [inline doc/examples/data/plot_3d.py:18]: I would tend to reverse the import order.

**Review by mkcor — COMMENTED** [review]
Wonderful! I will link to this example whenever I use the dataset in question in other examples.

File: doc/examples/data/plot_3d.py
Comment (emmanuelle) [inline doc/examples/data/plot_3d.py:7]:
```suggestion
3 spatial dimensions (to be distinguished from to 2D multichannel images, which 
are also arrays with
```

File: doc/examples/data/plot_3d.py
Comment (emmanuelle) [inline doc/examples/data/plot_3d.py:18]:
```suggestion
import plotly
import plotly.express as px
```

File: doc/examples/data/plot_3d.py
Comment (mkcor) [inline doc/examples/data/plot_3d.py:7]:
```suggestion
3 spatial dimensions (to be distinguished from 2D multichannel images, which 
```

File: doc/examples/data/plot_3d.py
Comment (mkcor) [inline doc/examples/data/plot_3d.py:26]: For my personal understanding, why isn't `fig.show()` enough in this case? I have checked locally that it isn't indeed, but I don't know why. :hushed: Thanks!

**Review by mkcor — APPROVED** [review]
Up to the tiny typo fix!

**Comment by mkcor** [comment]
The test fail is unrelated to this PR.

@jni I remember you mentioning that docs-only PR required only one approving review before merging? Under https://scikit-image.org/docs/stable/contribute.html#guidelines, I'm reading that "No changes are ever committed [to master] without review and approval by two core team members" so I'm hesitant... Should we update these guidelines?

File: requirements/docs.txt
Comment (grlee77) [inline requirements/docs.txt:18]: I wasn't familiar with `kaleido`, but it looks like plotly uses it for static image export when it is available.

**Review by grlee77 — APPROVED** [review]
Thanks, Emma. The [doc artifact for this looks good](https://1206-2014929-gh.circle-artifacts.com/0/doc/build/html/auto_examples/data/plot_3d.html#sphx-glr-auto-examples-data-plot-3d-py). Very convenient to have that in the PR now!

**Comment by grlee77** [comment]
> @jni I remember you mentioning that docs-only PR required only one approving review before merging? Under https://scikit-image.org/docs/stable/contribute.html#guidelines, I'm reading that "No changes are ever committed [to master] without review and approval by two core team members" so I'm hesitant... Should we update these guidelines?

I would be +1 on updating the guidelines there. One approving review seems sufficient for doc PRs. I think it is mainly PRs involving API changes where it is important to get consensus from multiple reviewers.

File: requirements/docs.txt
Comment (mkcor) [reply requirements/docs.txt:18]: Right, see https://plotly.com/python/static-image-export/.

**Comment by mkcor** [comment]
Ok, done! Thanks, @grlee77 -- will submit an update to the contribution guidelines.

**Pull request merged**
